src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -37,7 +37,8 @@ Copyright (c) .NET Foundation. All rights reserved.
   </PropertyGroup>
 
   <PropertyGroup>
-    <GenerateRuntimeConfigurationFiles Condition=" '$(GenerateRuntimeConfigurationFiles)' == '' and '$(TargetFrameworkIdentifier)' == '.NETCoreApp' and ('$(HasRuntimeOutput)' == 'true' or '$(UseComHost)' == 'true') ">true</GenerateRuntimeConfigurationFiles>
+    <IsComponent Condition="'$(IsComponent)' == '' and '$(UseComHost)' == 'true'">true</IsComponent>
```

Comment:
IsComponent feels a little off to me. There aren't a lot of "Is" properties in msbuild. And maybe "component" is too abstract?

Reply:
@KathleenDollard @richlander for help on naming this.

Reply:
Definitely would like better naming suggestions.

Reply:
The first name I had was `BuildAsComponent`, but that seemed even less consistent with other known properties.

Maybe another option could be to make the property indicate something like `ComponentType` and have different known values for basic component / plugin / com server?

Reply:
@KathleenDollard @richlander I would appreciate any help you could provide on naming.

Reply:
Per offline discussion, will be changing:
`IsComponent` -> `EnableDynamicLoading`
`UseComHost` -> `EnableComHosting`